scripts/tests/test_phase42_boundary_correspondence.py

```
@@ -0,0 +1,415 @@
+        phase29 = sample_phase29_contract(hash32("d"), hash32("e"))
+        phase30 = sample_phase30_manifest(hash32("7"), hash32("8"))
+        phase41 = PHASE42.prepare_phase41_expected(phase29, phase30)
+        with tempfile.TemporaryDirectory() as tempdir:
+            temp = pathlib.Path(tempdir)
+            phase29_path = temp / "phase29.json"
+            phase30_path = temp / "phase30.json"
+            phase41_path = temp / "phase41.json"
+            phase29_path.write_text(json.dumps(phase29), encoding="utf-8")
+            phase30_path.write_text(json.dumps(phase30), encoding="utf-8")
+            phase41_path.write_text(json.dumps(phase41), encoding="utf-8")
+            completed = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)